File: 204/204.py
#Low Memory
#Runtime: 11216 ms
class Solution:
    def countPrimes(self, n: 'int') -> 'int':
        def isPrime(n):
            if n <= 3:
                 return n > 1
            else:
                for i in range(2, int(math.sqrt(n)) + 1):
                    if n%i == 0:
                        return False
            return True
        sum = 0
        for i in range(1, n):
            if isPrime(i):
                sum += 1
        return sum

File: 204/204.java
class Solution {
    public static boolean isPrime(int n){
            if (n <= 3)
                 return n > 1;
            else{
                for(int i=2;i<=Math.sqrt(n);i++){
                    if(n%i == 0)
                        return false;
                }
            }
            return true;
          }
    public int countPrimes(int n) {
        n -= 1;
        int sum = 0;
        for (int i = 1; i <= n; i++){
             if (isPrime(i)) {
                sum++;
             }
        }
        return sum;
    }
}
